lib/features/Notification.dart/widgets/notification_card.dart

```diff
@@ -0,0 +1,61 @@
+
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:quranpally/config/asset/app_assets.dart';
+import 'package:quranpally/shared/widgets/app_image.dart';
+
+class NotificationCard extends StatelessWidget {
+  const NotificationCard({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      margin: const EdgeInsets.only(bottom: 20),
+      height: 50.h,
+      width: 390.w,
+      child: Row(
+        children: [
+          AppImageWidget(
+            reference: AppAssets.userImage,
```

Comment:
keep in mind we will change this latter to fetch userImage from remote source i.e (https)